goon/browserassets/js/browserOutput.js

```diff
@@ -86,7 +86,7 @@ if (typeof String.prototype.trim !== 'function') {
 
 //Shit fucking piece of crap that doesn't work god fuckin damn it
```

Comment:
Petition to run a script which appends this to the start of every .DM file in our codebase.